Examples/eplain wg filter (#2421)
pyansys/pyaedt: doc/source/API/SetupTemplatesIcepak.rst

Icepak templates and arguments
===============================

This section lists all setup templates with their default values and keys available in Icepak.

> Please note that Icepak parameters contains spaces. In order to use them as arguments of ``"create_setup"`` method

Reviewer: ```suggestion
Note that Icepak parameters contain spaces. To use them as arguments of the ``"create_setup"`` method, these
```
